### PR TITLE
Complete are_all_terminal_cool_sources_chilled_water

### PR DESCRIPTION
**Pull request by weilixu**

### CODE REVIEW

**Review by yunjoonjung-PNNL — APPROVED** [review]
Looks great to me. Thank you!

File: rct229/ruleset_functions/baseline_systems/baseline_hvac_sub_functions/are_all_terminal_cool_sources_chilled_water.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        List of terminal units IDs
+    Returns
+    -------
+    bool
+        True: cool source associated with all terminal units sent to this function is CHILLED_WATER
+        False: any terminal unit has a cool source other than CHILLED_WATER
+    """
+    are_all_terminal_cool_sources_chilled_water_flag = True
+    for terminal_b_id in terminal_unit_id_list:
+        terminal_b = find_exact_one_terminal_unit(rmi_b, terminal_b_id)
```
Comment (charliepnnl) [inline rct229/ruleset_functions/baseline_systems/baseline_hvac_sub_functions/are_all_terminal_cool_sources_chilled_water.py:24]: I'd like to suggest that we shorten the function names for these functions by removing the `exact_one_`.  It seems clear enough that we should expect to find exactly one.

File: rct229/ruleset_functions/baseline_systems/baseline_hvac_sub_functions/are_all_terminal_cool_sources_chilled_water.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        if terminal_b.get("cooling_source") != COOLING_SOURCE.CHILLED_WATER:
```
Comment (charliepnnl) [inline rct229/ruleset_functions/baseline_systems/baseline_hvac_sub_functions/are_all_terminal_cool_sources_chilled_water.py:25]: Would the intent be clearer with the function body:
```
return all(
      find_terminal_unit(rmi_b, terminal_b_id).get("cooling_source")
      == COOLING_SOURCE.CHILLED_WATER
      for terminal_b_id in terminal_unit_id_list
)
```
?

**Review by charliepnnl — APPROVED** [review]
Approved with a couple of style comments.

**Pull request merged**